### ISSUE TITLE
Enable and Disable DNS rewrites

### ISSUE THREAD

**Issue by Archive5** [AdguardTeam/AdGuardHome#1765]
#### Description

Add column with check boxes for enable and disable each DNS rewrites same as blocklists

![image](https://user-images.githubusercontent.com/63988538/83607114-7ef12400-a52f-11ea-8243-76269463b797.png)

**Comment by devinslick** [comment]
@ameshkov ,
Is there, or could there be a way to do this on a per-client basis?

**Comment by devinslick** [comment]
@ameshkov , I'm using rewrites to support internal records for a split zone.  Adding client granularity could allow only trusted clients to point to internal services.   Even a feature in the client settings like "Ignore all DNS rewrites" would be great for that.

**Comment by ameshkov** [comment]
@devinslick this may help then: https://github.com/AdguardTeam/AdGuardHome/issues/1761

Once $client modifier is added, "exclusion" rules may be added on a per-client basis, something like `@@||example.local$client=192.168.1.123`.

**Comment by mHatsune** [comment]
https://github.com/AdguardTeam/AdGuardHome/issues/1558

**Comment by KizunaH** [comment]
#2322

**Comment by brightdroid** [comment]
I've a similar/same problem, I'd like to rewrite a (internal) domain for all clients except my iot network.
This custom filter rule does not work:
```
||home.local^$client=192.168.100.0/23
```

**Comment by EugeneOne1** [comment]
Hello, @Aikatsui.  I'm glad to announce that we've finally implemented this feature.  It's already available in the edge-channel build, and it will also be included in the upcoming release.

**Comment by naxels** [comment]
Don't know if this is the right place to report,
However I just updated to servers to the new version and the Edit button is 'broken' as the popup for editing an existing entry shows blank input boxes. I can confirm that before the upgrade on at least one server it did work (did that before upgrading test after discovering the issue on the first upgraded server)

Also on a mobile device the edit/delete buttons are now a little misaligned but not a big deal

**Comment by ainar-g** [comment]
@naxels, yeah, it's a real (and rather shameful) bug.  Could you please file a new issue?  We'll try to fix it asap.

**EDIT:** #8072 has now been filled.
